novice-goals.Rmd

```diff
@@ -0,0 +1,201 @@
+-   What *isn't* included?
+    -   interactive plots
+    -   maps
+    -   3D visualization
+
+## Data analysis
+
+-   How can I represent and manage missing values?
+    -   NA
+-   How can I get a feel for my data?
+    -   summary statistics
+-   How can I create a simple model of my data?
+    -   formulas
+    -   linear regression
+-   How can I put people at risk?
+    -   algorithmic bias
+    -   de-anonymization
+-   What *isn't* included?
+    -   statistical tests
+    -   anything with "machine learning" in its name
+
+## Verification
+
+-   What is the difference between validation and verification?
+    -   did we build the right thing vs. did we build it correctly
+-   How can I make my programs tell me that something has gone wrong?
+    -   assertions
+-   How can I write and manage tests for my software?
+    -   unit tests
+    -   test runners
+-   What tests should I write for my software?
+    -   edge cases
+    -   fails when it should
+    -   bugs become tests
+-   What *isn't* included?
+    -   legal liabilities
+
+## Version control
+
+-   What is a version control system?
+    -   a smarter kind of backup
+-   How do I track my work locally?
+    -   diff
+    -   add
+    -   commit
+    -   log
+-   How do I view or recover an old version of a file?
+    -   diff
+    -   checkout
+-   How do I save work remotely?
+    -   push and pull
+-   How do I manage conflicts?
+    -   merge
+-   What *isn't* included?
+    -   forking
+    -   branching
+    -   pull requests
+    -   git reflow --substantive --single-afferent-cycle --ia-ia-rebase-fhtagn ...
+
+## Publishing
+
+-   How do static websites work?
+    -   URLs
+    -   servers
+    -   request/response cycle
+    -   pages
+-   How do I create a simple HTML page?
+    -   head/body
+    -   basic elements
+    -   images
+    -   links
+    -   relative vs. absolute paths
+-   How can I create a simple website?
+    -   GitHub pages
+-   How can I give pages a standard appearance?
+    -   layouts
+-   How can I avoid writing all those tags?
+    -   Markdown
+-   How can I share values between pages?
+    -   flat per-site and per-page configuration
+    -   variable expansion
+-   What *isn't* included?
+    -   templating
+    -   filters
+    -   inclusions
+
+## Reproducibility
+
+-   How can I make programs easy to read?
+    -   coding style
+    -   documentation
+-   How can I make programs easy to re-use?
+    -   writing functions
+    -   Taschuk's Rules
+-   How can I combine explanations, code, and results?
+    -   notebooks
+-   How should I organize my projects?
+    -   Noble's Rules
+-   How should I keep track of my data?
+    -   simple manifests
+-   What *isn't* included?
+    -   build tools (Make and its kin)
+    -   continuous integration
+    -   documentation generators
+
+## Collaboration
+
+-   What kinds of licenses are there?
+    -   open vs. closed
+    -   copyright
+-   What license should I use for my publications?
```

Comment:
Even before this, "Who gets to decide what license to use?" since some organizations, granting agencies, etc may have restrictions

Reply:
:+1: